docs/xrsonrestoresynconrestore.md

```diff
@@ -0,0 +1,55 @@
+﻿<!--
+* Copyright (c) 2017, 2023 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] https://openjdk.org/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -Xrs:[onRestore|syncOnRestore]
+
+**(Linux&reg; x86, Linux on POWER LE&reg;, and Linux on IBM Z&reg; only)**
+
+The `-Xrs` option is an existing option and `onRestore` and `syncOnRestore` are its new parameters that can be used for controlling the signal handling at the time of restore if the CRIU feature is available.
```

Comment:
Usual comment re passive voice. Would be better to say more about -Xrs. Eg "The -Xrs option is an existing option for disabling signal handling in the VM. CRIU support adds new suboptions `onRestore` and `syncOnRestore` for disabling signal handling when you restore a VM." Then you can remove the sentence about "if the CRIU feature is available", especially as that's noted later as a restriction anyway.